react/week2/todolist/src/Header.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from "react";
+
+function Header() {
+  const [timer, setTimer] = useState(0);
+
+  useEffect(() => {
+    const interval = setTimeout(() => {
+      setTimer((prev) => prev + 1);
+    }, 1000);
+    return () => clearTimeout(interval);
+  });
```

Comment:
Why do you need this : return () => clearTimeout(interval); ?
if you remove it then also ur timer works as required.